docs/components/connectors/use-connectors/index.md

```diff
@@ -75,6 +95,16 @@ To ensure process isolation, note that Connectors do not have access to process
 While using this field, a process variable with the name `response` is reserved.
 :::
 
+#### Example
+
+If you set `{ "bodyReceived": body }` inside the `Result Expression` field of the REST outbound Connector, this variable is available:
+
+```json
+{
+  "bodyReceived": "<h1>Hello, World!</h1>"
```

Comment:
See example above. HTML is not the best example as it cant be processed properly in comparison to JSON.